mantidproject/mantid
# Scale the About dialog box based on screen size - Needs Multiple testers

**Problem:**
The about box could appear tiny on high res small laptop screens.

**Description of work.**
see commits.
TLDR:
refactored the About dialog box into code from QI file and made it scale based on the screen it appears on.

**To test:**
This should ideally be tested on (stick a screenshot in the comments, with os & screen resolution):
- [x] RHEL or CENTOS
- [x] A high res laptop screen

1. open Mantid and help->about if it does not appear automatically
1. If you have more than one screen check it with Mantid on both at the time the help-about is clicked
1. check the links etc still work.

Windows 2560 * 1440
![image](https://user-images.githubusercontent.com/1017173/84684344-71289f00-af30-11ea-9a44-5de1e0d0b515.png)

Windows 1920 * 1080
![image](https://user-images.githubusercontent.com/1017173/84684296-640bb000-af30-11ea-9e6d-5ffbb3ded99b.png)

fixed pixel size .ui based version before these changes
![image](https://user-images.githubusercontent.com/1017173/84684110-198a3380-af30-11ea-889c-68c9074036f7.png)

Fixes #28789

*This does not require release notes* because **the dialog is new for this release**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
